recipes/sagemaker-code-editor/build.sh

```diff
@@ -2,15 +2,64 @@
 
 set -exuo pipefail
 
+# This code includes code from https://github.com/conda-forge/openvscode-server-feedstock
+# which is licensed under the BSD-3-Clause License.
+
+rm $PREFIX/bin/node
+ln -s $BUILD_PREFIX/bin/node $PREFIX/bin/node
```

Comment:
Why do we need to create this symlink from the host to the build environment for nodejs?

Reply:
Removed this since we're now specifying a fixed version of nodejs for build and host requirements